command/agent_test.go

```diff
@@ -2624,10 +2615,7 @@ func TestAgent_Metrics(t *testing.T) {
 
 	// Start a vault server
 	logger := logging.NewVaultLogger(hclog.Trace)
```

Comment:
We can probably move this down to the `testAgentCommand` invocation so it's more apparent what it's for.

Reply:
Done.